pkg/scheduler/generic_scheduler.go

```diff
@@ -428,10 +428,12 @@ func prioritizeNodes(
 		return nil, scoreStatus.AsError()
 	}
 
-	if klog.V(10).Enabled() {
+	// Additional details logged at level 10 if enabled.
+	klogV := klog.V(10)
```

Comment:
This gets used again below and therefore is not scoped to just the following if clause.